cinema_service/settings.py

```diff
@@ -140,3 +137,37 @@
 # https://docs.djangoproject.com/en/4.0/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+REST_FRAMEWORK = {
+    "DEFAULT_PERMISSION_CLASSES": [
+        "cinema.permissions.IsAdminOrIfAuthenticatedReadOnly"
```

Comment:
The `DEFAULT_PERMISSION_CLASSES` setting uses a custom permission class `IsAdminOrIfAuthenticatedReadOnly`. Ensure that this class is implemented correctly to enforce the desired access control.